consensus/hotstuff/signature/signature_aggregator_test.go

```diff
@@ -0,0 +1,79 @@
+package signature
+
+import (
+	"fmt"
+	"testing"
+
+	mock2 "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/module/mock"
```

Comment:
```suggestion
	testifymock "github.com/stretchr/testify/mock"
	"github.com/stretchr/testify/require"

	"github.com/onflow/flow-go/crypto"
	modulemock "github.com/onflow/flow-go/module/mock"
```
Clearer aliases than the default for the conflicting import names